src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -311,10 +308,6 @@ private boolean perform(Run<?, ?> run,
     private String getRootUrl() {
         Jenkins instance = Jenkins.getInstance();
 
-        if (null == instance) {
```

Comment:
I think this change is ok.
`Jenkins::getInstance` is now annotated with `Nonnull` and this code only runs when Jenkins is completely initialized.

Reply:
Yup, and also spotbugs doesn't give you much choice on this. Without this change and the one below, `mvn install` wouldn't complete successfully.